datafusion-cli/src/main.rs

```diff
@@ -121,6 +121,13 @@ struct Args {
     )]
     maxrows: MaxRows,
 
+    #[clap(
+        short,
+        long,
+        help = "Whether to stop early when max rows is reached, this will help reduce the memory usage when the result is large"
```

Comment:
I think this option is basically the same as adding a `LIMIT max_rows` to the query and thus is somewhat redundant

Reply:
Very good point, remove it now.

Reply:
The option appears to still be present